app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -0,0 +1,28 @@
+package org.simple.clinic.home.overdue
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.RelativeLayout
+
+class OverdueScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
+
+
+  companion object {
```

Comment:
I've had a discussion with @saket about this, and this is something we should finalize. Companion objects are used mainly to interop between Java and Kotlin code, and my opinion is that exposing a constant `KEY` has no value.

My take is that we should just construct an instance of the key whenever we need to load the screen instead of using this static constant.